docs/reference/security/fips-140-compliance.asciidoc

```diff
@@ -8,59 +8,75 @@ government computer security standard used to approve cryptographic modules.
 {es} offers a FIPS 140-2 compliant mode and as such can run in a FIPS 140-2
 configured JVM.
 
-IMPORTANT: The JVM bundled with {es} is not configured for FIPS 140-2. You must 
+IMPORTANT: The JVM bundled with {es} is not configured for FIPS 140-2. You must
 configure an external JDK with a FIPS 140-2 certified Java Security Provider.
 Refer to the {es}
 https://www.elastic.co/support/matrix#matrix_jvm[JVM support matrix] for
-supported JVM configurations.
+supported JVM configurations. See https://www.elastic.co/subscriptions[subscriptions] for required licensing.
 
-After configuring your JVM for FIPS 140-2, you can run {es} in FIPS 140-2 mode by
-setting the `xpack.security.fips_mode.enabled` to `true` in `elasticsearch.yml`.
+Compliance with FIPS 140-2 is achieved by using only FIPS approved / NIST recommended cryptographic algorithms. This can be achieved by the following:
```

Comment:
I'm not sure we want to make definitive statements about to achieve "compliance". 
For example, what about the Operating System? Container runtime? Hardware crypto devices?  

I'd be more comfortable stating either:
1. A negative "You are not in compliance unless... "
2. A statement that appears less exhaustive "Compliance requires that ..."

The old wording was problematic as well, but at least it was prefixed with `For {es}, ...`

Reply:
++, how is this : https://github.com/elastic/elasticsearch/pull/105041/commits/4b1855f6d7a2d299125693eeb7fb38a0499f630a ?

Reply:
LGTM